btree/btree.c

```diff
@@ -1921,12 +1951,20 @@ static bool segaddr_is_valid(const struct segaddr *seg_addr)
 static struct segaddr segaddr_build(const void *addr, int shift)
 {
 	struct segaddr sa;
+#if 0
```

Comment:
Remove #if 0 code wherever present in this file.

Reply:
Done.